pkg/controller/operators/catalog/operator.go

```diff
@@ -903,6 +904,17 @@ func (o *Operator) syncResolvingNamespace(obj interface{}) error {
 		return err
 	}
 
+	failForwardEnabled := false
+	ogs, err := o.lister.OperatorsV1().OperatorGroupLister().OperatorGroups(namespace).List(labels.Everything())
```

Comment:
not sure if the diff changed here, but listers query the informer cache; they don't even contain an http client

Reply:
> I ran this locally and ran into the following error (after added a debug level log and re-deploying the catalog operator):
> 
> ```
> time="2022-04-14T13:54:20Z" level=debug msg="failed to list the operatorgroups in the ff-e2e-fcprn namespace: cannot list OperatorGroups with a NullOperatorGroupNamespaceLister"
> ```
> 
> /hold

We don't have OG lister on catalog operator. That's why I said in one of my comments about adding the informer and lister for operatorgroup in order to use the OperatorGroupLister here. (See like 887 on catalog/operator.go).

Reply:
I meant there is currently no informer or lister for OG in catalog operator. You will need to registry it. Otherwise, you will need to use k8s client to query OG directly. My original comment was a bit confusing.